# Generated by Django 2.2.16 on 2021-03-19 17:07

from django.db import migrations


class Migration(migrations.Migration):

    dependencies = [
        ('error', '0008_auto_20210319_1448'),
    ]

    operations = [
        migrations.AlterModelOptions(
            name='dataepidemiologiaerror',
            options={'verbose_name': 'Error de procesamiento de archivo (Epidemiología)', 'verbose_name_plural': 'Errores de procesamiento de archivo (Epidemiología)'},
        ),
        migrations.AlterModelOptions(
            name='datahigieneerror',
            options={'verbose_name': 'Error de procesamiento de archivo (Higiene)', 'verbose_name_plural': 'Errores de procesamiento de archivo (Higiene)'},
        ),
        migrations.AlterModelOptions(
            name='placaerrormodel',
            options={'verbose_name': 'Error de procesamiento de la placa', 'verbose_name_plural': 'Errores de procesamiento de las placas'},
        ),
    ]
